Cargo.toml

```diff
@@ -57,4 +59,8 @@ members = [
         ".",
         "examples/postgres/pooled-with-rustls",
         "examples/postgres/run-pending-migrations-with-rustls",
+        "examples/generic-async",
 ]
+
+[patch.crates-io]
+diesel = { path = "../diesel/diesel" }
```

Comment:
Note to self: change this once diesel is updated with needed changes or released with them

Reply:
For now we can also use a git dependency to your branch here. That we at least the CI can run and test something.

Reply:
Added in a specific commit so it's easier to remove